pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages:
```

Comment:
Yes, specific examples are often helpful, e.g. for demonstrating the expected format. In this case if you think a concrete example would be more helpful, by all means go ahead.